packages/physics-physx/src/Collider.ts

```diff
@@ -0,0 +1,80 @@
+import { ICollider } from "@oasis-engine/design";
+import { Quaternion, Vector3 } from "@oasis-engine/math";
+import { ColliderShape } from "./shape/ColliderShape";
+
+/**
+ * physical collider
+ */
+export abstract class Collider implements ICollider {
```

Comment:
Rename `Collider` to `PhysxCollider`, same as other files.

Try not to repeat the class name in the official Oasis package.

Reply:
done